client/src/www/model/platform_error.ts

```diff
@@ -47,37 +52,71 @@ function createInternalError(cause?: unknown): PlatformError {
  * @returns {PlatformError} A non-null instance of PlatformError.
  * @throws {Error} Will be thrown when {@link rawObj} is invalid.
  */
-function convertRawErrorObjectToPlatformError(rawObj: object): PlatformError {
+function convertRawErrorObjectToError(rawObj: object): Error {
```

Comment:
Replace the comment in line 50 and 52 from`{@link PlatformError}` to `Error`

Reply:
Done